core/domain/html_validation_service.py

```diff
@@ -707,8 +707,8 @@ def is_parsable_as_xml(xml_string):
 
 def convert_svg_diagram_tags_to_image_tags(html_string):
     """Renames all the oppia-noninteractive-svgdiagram on the
-       server to oppia-noninteractive-image and changes
-       corresponding attributes
+    server to oppia-noninteractive-image and changes
```

Comment:
The line wrapping seems a bit off here. Please update the docstring to use as many words as possible up to the line limit (80 characters) before wrapping. I think 'server to' can fit on the first line, for instance.

Reply:
done.